packages/sdk-metrics/test/Instruments.test.ts

```diff
@@ -374,6 +380,9 @@ describe('Instruments', () => {
           unit: '',
           type: InstrumentType.HISTOGRAM,
           valueType: ValueType.INT,
+          advice: {
+            explicitBucketBoundaries: [1, 9, 100],
+          },
```

Comment:
The previous test should have this but left it out due to it not being enforced by the type. Internally that went to `assertPartialDeepStrictEqual` and that was only checking everything appears here (`expected`) is present and `===` on the `actual`, but not the other way around.